debugger/ptraceLib/enclave_context.c

```diff
@@ -224,8 +223,25 @@ static int _get_enclave_thread_current_ssa_info(
         return -1;
     }
 
+    // SSA is assigned to the page immediately after the tcs page.
+    // SSA address is also derived from adding tcs.ossa to enclave base address.
+    // See: host/sgx/create.c
```

Comment:
+ @radhikaj For scheduling/decision about contributing upstream vs separating it out.

To me separating it out would be a good first step nonetheless as it would avoid accidental dependencies with rest of OE.
Two pending tasks to make this module separable/upstreamable are:
- Remove dependence on OE headers
- Document the binary contract - especially the following two assumptions
  - The OSSA page must allocated immediately after TCS page.
  - The byte offset at which to fetch __ssa_frame_size from td_t structure.

@radhikaj Let me know what the scheduling is; I can take a stab at moving this to a separate repo that would help upstream later.

We can sanitize the comments as part of that effort.